op-node/rollup/finality/finalizer_test.go

```diff
@@ -468,3 +532,21 @@ func TestEngineQueue_Finalize(t *testing.T) {
 		emitter.AssertExpectations(t)
 	})
 }
+
+func helperL2BlockRefByNumberWithTimes(t *testing.T, l2F *testutils.MockL2Client, times int, refs ...eth.L2BlockRef) {
```

Comment:
is `t` used anyway? same for `helperSdkClientQueryBlockRangeBabylonFinalized`

Reply:
no, it should be removed